openapi/coffeecard_api_v2.swagger.json

```diff
@@ -1,511 +1,1135 @@
 {
+
```

Comment:
There's clearly something wrong with your local `json` formatter with all these extra lines added. This file should be reset and re-added without all those changes.

Reply:
formatting fixed it. should be good now